Odata-docs/docfx.json

```diff
@@ -3,8 +3,8 @@
     "content": [
       {
         "files": [
-          "**/*.md",
-          "**/*.yml"
+          "**/*.{md,yml}",
```

Comment:
this line should be `"*.{md,yml}"`

Reply:
I think I had it that way a few versions back, but changing to see